frontend/app/src/layouts/PreviewLayout/index.tsx

```diff
@@ -1,22 +1,34 @@
-import React, { FC, ReactNode } from 'react';
-import { PreviewLayoutContainer } from '@popr/ui';
-import { ContactList } from '@popr/ui';
-import { getContacts } from 'services/contacts';
-import { useQuery } from 'react-query';
-import { useTranslation } from 'react-i18next';
+import React, { FC, ReactNode, useState } from 'react';
+import { PreviewLayoutContainer, Slide } from '@popr/ui';
+import AddIcon from '@mui/icons-material/Add';
+import { GeneralIconButton } from '@popr/ui';
 
 type PreviewLayout = {
   children?: ReactNode;
 };
 
 const PreviewLayout: FC<PreviewLayout> = () => {
-  const { t } = useTranslation(['config', 'translations']);
-  const { data: contacts } = useQuery(['getContacts'], async () => await getContacts('63120fd98028831ecf43b444'));
+  const exampleSlide = { header: 'header', main: 'some text', footer: 'footer' };
+  const [slides, setSlides] = useState([exampleSlide]);
+
+  const addSlide = () => {
+    setSlides([...slides, exampleSlide]);
+  };
+
+  const showSlides = () => {
```

Comment:
Move this to a  component called SlideList. You can define it in app project.  Move all Slide logic to that componenr